blocks/footer/footer.css

```diff
@@ -1 +1,179 @@
 /* Empty file for now */
+footer {
+  background-color: #f6f6f6;
+}
+
+.footer {
+  padding-top: 35px;
+  padding-bottom: 15px;
+  font-family: var(--font-family-proxima);
+  color: #3a3a3a;
+  line-height: 23px;
+}
+
+.footer-container {
+  max-width: 1350px;
+  margin: 0 auto;
+  width: 100%;
+  padding: 0 15px;
+}
+
+.footer-container-flex {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.footer-container-flex > div:not(:last-child)  {
+  position: relative;
+  width: 100%;
+  min-height: 1px;
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+.column-12 {
+  -webkit-box-flex: 0;
+  -webkit-flex: 0 0 100%;
+  -ms-flex: 0 0 100%;
+  flex: 0 0 100%;
+  max-width: 100%;
+}
+
+.link-menu-row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -15px;
+  margin-left: -15px;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.link-menu-col {
+  position: relative;
+  width: 100%;
+  min-height: 1px;
+  padding-right: 15px;
+  padding-left: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.link-menu-col ul {
+  list-style-type: none;
+}
+
+.link-menu-col li {
+  padding: 6px 0;
+  letter-spacing: .4px;
+  font-size: 12px;
```

Comment:
Use one of the font size variables here, so that if we need to change it, it's adjusted in concert with other locations.